distribution/tools/launchers/src/test/java/org/elasticsearch/tools/launchers/JvmOptionsParserTests.java

```diff
@@ -149,6 +156,108 @@ public void testComplexOptions() throws IOException {
         }
     }
 
+    public void testMissingRootJvmOptions() throws IOException, JvmOptionsParser.JvmOptionsFileParserException {
+        final Path config = newTempDir();
```

Comment:
This is what is triggering the need for the temporary directory to exist.

Reply:
I can't seem to find any other usages of this method in any of our other tests. Perhaps it relies on some different assumptions regarding test execution environment. Our other tests seem to rely on `LuceneTestCase.createTempDir()` instead.

This project doesn't seem to use our test framework, just the "vanilla" randomized runner stuff.

Reply:
Yeah, and that ensures that the temporary directory exists: https://github.com/apache/lucene-solr/blob/9a4f7661e96416e3aac3e48f1108422ab4184473/lucene/test-framework/src/java/org/apache/lucene/util/TestRuleTemporaryFilesCleanup.java#L187